src/components/TaskBoard/index.jsx

```diff
@@ -0,0 +1,29 @@
+import { useState } from "react";
+import TaskList from "/src/components/TaskList";
+import AddTask from "/src/components/Addtask";
+import "./index.scss"
+
+const TaskBoard = () => {
+  const [isFormOpen, setIsFormOpen] = useState(false);
+
+  const toggleForm = () => {
+    setIsFormOpen(!isFormOpen);
+  };
+
+  return (
+    <div className="task-board">
+      <div className="task-board__create-button-container margin-bottom">
+        <button onClick={toggleForm}>
```

Comment:
[reference](https://react.dev/learn/responding-to-events)
![Screenshot 2023-08-02 at 5 34 38 PM](https://github.com/hoh-wtag/react-todo/assets/128278816/356b3146-fc7e-4b3b-8846-caa43dd4e779)